RHDEVDOCS-4980: Update the pipelines 1.7 RN (#55866)
openshift/openshift-docs: modules/op-release-notes-1-7.adoc

** To add the annotations to a specific namespace, export the cluster task as a task to that namespace.
// https://issues.redhat.com/browse/SRVKP-1514

> * Before this update, the `when` expression would skip a `Task` object and its dependent tasks when certain conditions were not met. With this update, you can scope the `when` expression to guard the `Task` object only, not its dependent tasks. To enable this update, set the `scope-when-expressions-to-task` flag to true in the `TektonConfig` CRD.

Reviewer: Do we need to quote true? set the `scope-when-expressions-to-task` flag to `true` in the `TektonConfig` CRD.?

Author: @VeereshAradhya we might remove the last sentence, but the intent is to let user know how he can enable this new update. If we have mentioned this part somewhere in the Pipelines documentation, then we can go away with this last statement.

Reviewer: I just wanted to ask if we want to write true or `true` in the last sentence

Reviewer: Instead of "To enable this update, set the `scope-when-expressions-to-task` flag to true in the `TektonConfig` CRD" could we write "To enable this update, set the `scope-when-expressions-to-task` flag to `true` in the `TektonConfig` CRD"?

> * Before this update, the `when` expression would skip a `Task` object and its dependent tasks when certain conditions were not met. With this update, you can scope the `when` expression to guard the `Task` object only, not its dependent tasks. To enable this update, set the `scope-when-expressions-to-task` flag to `true` in the `TektonConfig` CRD.

Reviewer 2: Maybe, for clarity:
```suggestion
* Before this update, if certain conditions were not met, the `when` expression skipped a `Task` object and its dependent tasks. With this update, you can scope the `when` expression to guard the `Task` object only, not its dependent tasks. To enable this update, set the `scope-when-expressions-to-task` flag to `true` in the `TektonConfig` CRD.
```
- Follows the "if...when" pattern: puts the triggering condition first.
- Avoids repeating different instances of "when" in the same sentence.

Author: Incorporated this suggestion @rolfedh

> * This update adds a new flag named `scope-when-expressions-to-task`. Before this update, the `when` expression would skip a `Task` object and its dependent tasks when certain conditions were not met. With this update, you can scope the `when` expression to guard the `Task` object only, not its dependent tasks. To enable this update, set the `scope-when-expressions-to-task` flag to true.

Reviewer 3: Is 1.7 the first time the `scope-when-expressions-to-task` appear ? 🤔 
The rest is good, and the following should be added as well.

> The `scope-when-expressions-to-task`flag is deprecated and will be removed in a future release. As a best practice for {pipelines-shortname}, use`when`expressions scoped to the guarded`Task` only.

Author: @vdemeester, Removed the first sentence to avoid any ambiguity and kept the rest of the information as is. Please verify and confirm. Thanks
